### ISSUE TITLE
Got the same tweets

### ISSUE THREAD

**Issue by JiayingCai9** [taspinar/twitterscraper#185]
The number of tweets I got was larger than the actual number.
I searched it and found there were only 22 tweets, but the code returned 41. As I scraped the users' information, I compared them and found tweets were saved repeatedly.

This is my code.
list_of_tweets = query_tweets("Tesla Sale", limit=None, begindate=dt.date(2019, 3, 13), enddate=dt.date(2019, 3, 14), poolsize=20, lang='')

The results are like:

<img width="604" alt="屏幕快照 2019-05-22 下午7 50 23" src="https://user-images.githubusercontent.com/50069420/58172335-e47ac580-7cca-11e9-838c-e7b9b8ca0433.png">

**Comment by AntonioNieto92** [comment]
same problem, got the same tweets repeated twice

**Comment by chaojiang06** [comment]
same thing happened to me, is there any solution?

Thanks!

**Comment by renatomello** [comment]
The same issue is happening to me as well

**Comment by StormFalcon32** [comment]
Yeah I'm getting duplicate tweets. Even worse, it doesn't always happen. The easiest way is to probably just save all of them and then process out duplicates later
